internal/rootcoord/import_manager.go

```diff
@@ -369,3 +367,73 @@ func (m *importManager) getTaskState(id int64) *milvuspb.GetImportStateResponse
 
 	return resp
 }
+
+// load Loads task info from Etcd when RootCoord (re)starts.
+func (m *importManager) load() error {
```

Comment:
The load() method should be called in the init() method:
func (m *importManager) init() error {
	//  read task list from etcd
        m.load()
	// trigger Import() action to DataCoord
	m.pushTasks()

	return nil
}

Reply:
Sure. I put it there.